File: python/scripts/get_local_summary.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
restriction = travelperk.travelsafe().travelsafe().local_summary(
    "ES", "country_code"
)
write_output("local_summary", [
    restriction.summary,
    restriction.details,
    restriction.risk_level.id,
    restriction.risk_level.name,
    restriction.risk_level.details,
    restriction.location.name,
    restriction.location.type,
    restriction.location.country_code,
    restriction.updated_at,
    restriction.guidelines,
    restriction.guidelines[0].category.id,
    restriction.guidelines[0].category.name,
    restriction.guidelines[0].sub_category.id,
    restriction.guidelines[0].sub_category.name,
    restriction.guidelines[0].summary,
    restriction.guidelines[0].details,
    restriction.guidelines[0].severity,
])

File: php/scripts/get_local_summary.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$restriction = $travelperk->travelSafe()->travelSafe()->localSummary(
	"ES",
	"country_code",
);
write_output("local_summary", [
	$restriction->summary,
	$restriction->details,
	$restriction->riskLevel->id,
	$restriction->riskLevel->name,
	$restriction->riskLevel->details,
	$restriction->location->name,
	$restriction->location->type,
	$restriction->location->countryCode,
	$restriction->updatedAt,
	$restriction->guidelines,
	$restriction->guidelines[0]->category->id,
	$restriction->guidelines[0]->category->name,
	$restriction->guidelines[0]->subCategory->id,
	$restriction->guidelines[0]->subCategory->name,
	$restriction->guidelines[0]->summary,
	$restriction->guidelines[0]->details,
	$restriction->guidelines[0]->severity,
]);
